static/js/components/ClassificationEntry.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { useEffect, useState } from "react";
+import PropTypes from "prop-types";
+import { useSelector } from "react-redux";
+import { useForm, Controller } from "react-hook-form";
+import Checkbox from "@material-ui/core/Checkbox";
+import TextField from "@material-ui/core/TextField";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+import Typography from "@material-ui/core/Typography";
+import Button from "@material-ui/core/Button";
+import Box from "@material-ui/core/Box";
+
+import FormValidationError from "./FormValidationError";
+import styles from "./ClassificationEntry.css";
+
+const ClassificationEntry = ({ addClassification }) => {
+  const userGroups = useSelector((state) => state.groups.user);
```

Comment:
Groups are displayed for classifications like comments, but there is no option in the comments form to set group ownership. So everything is being shared by default?

Reply:
The idea for this first PR is to have classifications shared with all groups that the user/token belongs to. We can add fine grained classifications in another PR.